src/plugins/python/pythonplugin.cpp

```diff
@@ -363,6 +363,32 @@ QString PythonMapFormat::nameFilter() const
     return ret;
 }
 
+QString PythonMapFormat::shortName() const
+{
+    QString ret;
+
+    // find fun
+    PyObject *pfun = PyObject_GetAttrString(mClass, "shortName");
+    if (!pfun || !PyCallable_Check(pfun)) {
+        PySys_WriteStderr("Plugin extension doesn't define \"shortName\"\n");
+        return ret;
```

Comment:
Maybe it would be helpful to fall back to returning the result from `nameFilter` here, for backwards compatibility?